### ISSUE TITLE
Add support for combined error/helper text below inputs

### ISSUE THREAD

**Issue by cmeeren** [ButchersBoy/MaterialDesignInXamlToolkit#889]
I can't find any support for helper text below inputs as described [here in the Material Design documentation](https://material.io/guidelines/patterns/errors.html#errors-user-input-errors) (note its interplay with error/validation text). Would it be possible to add support for this?

**Comment by Insire** [comment]
I'm pretty sure thats already supported, when you bind to ViewModels that implement INotifyDataErrorInfo or IDataErrorInfo.

**Comment by cmeeren** [comment]
AFAIK that's just for errors, not helper text.

**Comment by Insire** [comment]
Ah, you mean the small gray caption below the input fields, that also turns red once a validation error occurs? I dont think there is currently support for that.

**Comment by cmeeren** [comment]
Yes, that is exactly what I'm talking about. :-) Note that the text need not be the same as any error message.

**Comment by ChrisLawford** [comment]
Second this! would be great if it is included in the future

**Comment by andrejpanjan** [comment]
I third that!

**Comment by MichelMichels** [comment]
I'm currently working to implement this.
